JetMETCorrections/Modules/interface/JetCorrectionESChain.h

```diff
@@ -1,33 +1,5 @@
 #ifndef JetCorrectionESChain_h
 #define JetCorrectionESChain_h
 
-//
-// Original Author:  Fedor Ratnikov
-//         Created:  Feb. 13, 2008
-//         (originally named JetCorrectionServiceChain, renamed in 2011)
-//
-//
 
-#include <string>
-#include <memory>
-#include <vector>
-
-#include "FWCore/Framework/interface/ESProducer.h"
-
-class JetCorrectionsRecord;
-class JetCorrector;
-namespace edm {
-  class ParameterSet;
-}
-
-class JetCorrectionESChain : public edm::ESProducer {
-public:
-  JetCorrectionESChain(edm::ParameterSet const& fParameters);
-  ~JetCorrectionESChain() override;
-
-  std::unique_ptr<JetCorrector> produce(JetCorrectionsRecord const&);
-
-private:
-  std::vector<std::string> mCorrectors;
-};
 #endif
```

Comment:
Why not simply removing the whole file?

Reply:
That was my intention, but apparently I forgot to actually do that. Thanks for pointing it out.